packages/developer-portal/src/components/apps/page/helper.tsx

```diff
@@ -287,6 +288,26 @@ export const Helper: FC = () => {
     )
   }
 
+  if (isAppsInstallations) {
+    return (
+      <div className={elFadeIn}>
+        <Icon className={elMb3} icon="myAppsInfographic" iconSize="large" />
+        <Subtitle>Apps Documentation</Subtitle>
+        <SmallText hasGreyText>
+          This is the dashboard for your applications created using the Reapit Foundations platform. If you have not
+          created an app before or you need help, please take the time to view our getting started guide.
+        </SmallText>
+        <Button className={elMb3} intent="neutral" onClick={openNewPage(ExternalPages.developerPortalDocs)}>
+          View Docs
+        </Button>
```

Comment:
Wasn't sure if different info was to be displated here?

Reply:
Maybe, I think it's fine for now - Holly can always suggest some new text later